document-translator/script.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import './pdf.min.mjs';
+
+const {pdfjsLib} = globalThis;
```

Comment:
Nit: I wonder how much value pdf support adds to this demo. It could be simpler/easier to read without it.

Reply:
It's an actual feature request, this happens to be a product and a demo. I'll CC: you into the thread.